### DISCUSSION TITLE
Support for DB migrations (Liquibase or Flyway)?

### DISCUSSION THREAD

**Discussion by sideeffffect** [oyvindberg/typo#32] [Q&A]
Hello, does this library work well (or even integrate) with DB migration frameworks like Liquibase or Flyway?

**Comment by oyvindberg** [comment]
Absolutely, but completely separately! 

You first run the process to apply migrations, then you run typo to regenerate code. How you want to do this - during development, on ci and so on - is completely up to you

**Comment by i10416** [comment]
When it comes to database migration, it would be great if typo can perform migration in a declarative way using SQL files as this Rust crate does: https://crates.io/crates/renovate. There is also https://atlasgo.io, but it uses HCL for schema definition.

**Comment by pinki840** [comment]
@ksourabh321 kindly reach out to me pnayak060331@gmail.com (regarding liquibase
)
